### PR TITLE
Implement consistency maintainable

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement consistency maintainable

Ticket: https://trello.com/c/31feif08

### CODE REVIEW

File: src/utils/number.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * @param number The number to format.
  * @returns A string representing the formatted number.
  */
-export const formatNumberCompact = (number: number): string => {
+export const Number = (number: number): string => {
```
Comment (thanhnguyen-agilityio) [inline src/utils/number.ts]: function naming should more clear, and check if `Number` is a keyword.

File: src/pages/NotFound/index.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import MainLayout from '../../layouts/MainLayout';
-import utils from '../../styles/modules/utils.module.css';
+import stylesUtils from '../../styles/modules/stylesUtils.module.css';
```
Comment (thanhnguyen-agilityio) [inline src/pages/NotFound/index.tsx]: Keep `utils.module.css`. I mean only in naming of import.

If need same naming with file, use `as`. 

```
import utils as styleUtils from '../../styles/modules/utils.module.css';
```

File: src/mock/productDetail.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import mainImages from '../assets/images/detail-small.jpg';
+import secondaryImages from '../assets/images/small-t-shirt.jpg';
+import { Product } from '../types/product';
+
+export const productDetails: Product = {
```
Comment (thanhnguyen-agilityio) [inline src/mock/productDetail.ts:5]: `productDetails` -> `product`

File: src/utils/number.ts (hunk 1 above)
Comment (thanhtrantruong47) [reply src/utils/number.ts]: Fixed

File: src/pages/NotFound/index.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/pages/NotFound/index.tsx]: Fixed

File: src/mock/productDetail.ts (hunk 3 above)
Comment (thanhtrantruong47) [reply src/mock/productDetail.ts:5]: Fixed

Comment (hungdo-agilityio) [reply src/mock/productDetail.ts:5]: @thanhtrantruong47 Please recheck the convention. Use MACRO_CASE for constant

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**